build-farm/platform-specific-configurations/windows.sh

```diff
@@ -158,6 +159,7 @@ then
     export HAS_AUTOCONF=1
     export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-openssl=fetched --enable-openssl-bundling"
     export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-freemarker-jar=/cygdrive/c/openjdk/freemarker.jar"
+    export BUILD_ARGS="${BUILD_ARGS} --with-msvcp-dll=C:\Windows\System32\msvcp140.dll --with-msvcr-dll=C:\Windows\System32\vcruntime140.dll"
```

Comment:
Based on the freemarker line, I'm going to assume we need cyg paths.

Reply:
Updated it.